### ISSUE TITLE
tensorflow 2.0.0 build error 

### ISSUE THREAD

**Issue by YingUK** [jkjung-avt/jetson_nano#20]
Hi, 
Thanks for sharing your work! 

I followed your instruction and tried to install tensorflow 2.0.0 on my jetson nano, however it gave below error after a long time compilation.

```
tensorflow/python/lib/core/bfloat16.cc:608:60: note:   no known conversion for argument 2 from '<unresolved overloaded function type>' to 'PyUFuncGenericFunction {aka void (*)(char**, const long int*, const long int*, void*)}'
Target //tensorflow/tools/pip_package:build_pip_package failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 52964.953s, Critical Path: 1102.32s
INFO: 5632 processes: 5632 local.
FAILED: Build did NOT complete successfully

```
my system setup is:
jetpack 4.3
tensorrt 6.0.1
cuda 10.0.326
python 3.7

Could you please suggest how can fix this issue?

**Comment by jkjung-avt** [comment]
I have seen a similar issue when building tensorflow-2.3.0 on Jetson Xavier NX.  That's why I created the [tensorflow-2.3.0.patch](https://github.com/jkjung-avt/jetson_nano/blob/master/tensorflow/tensorflow-2.3.0.patch) with fixes for the "PyUFuncGenericFunction" issue.

**Comment by YingUK** [comment]
Thanks for your reply. I assume this patch cannot fix tensorflow 2.0.0 issue that i mentioned above. Otherwise can I install tf 2.3.0 version in jetpack 4.3 using your script?

**Comment by jkjung-avt** [comment]
The patch is for 2.3.0.  But you should be able to find the same lines of code in tensorflow-2.0.0 source and manually patch the code.

I have not tried to run "install_tensorflow-2.3.0.sh" on JetPack-4.3.  It might not work out of the box...

**Comment by YingUK** [comment]
Hi, 

I burned another SD card with JetPack 4.4, and installed tensorflow-2.3.0 successfully, although it took 3 days to complete on Nano. When it was installing scipy==1.4.1 as part of tensorflow wheel, it gave an error"numpy.disutils.system_info.NotFounfError: No lapacl/blas resources found". I fixed this issue based on [this discussion](https://github.com/scipy/scipy/issues/9005). solution is:
```
sudo apt-get install gfortran libopenblas-dev liblapack-dev
sudo pip install scipy==1.2.1
```
Hope this will be helpful to who may see this issue in the future. 
@jkjung-avt , thanks again for your work, I will close this issue for now.
